plexus-compiler-its/pom.xml

```diff
@@ -39,6 +39,8 @@
               <goal>verify</goal>
             </goals>
             <configuration>
+              <!-- If you want to run a single IT, use 'invokerTest' -->
```

Comment:
please remove, The best option is to use cli with `-Dinvoker.test=`

Reply:
I can remove it, but I put it there on purpose, because like many other developers, usually I just run Maven builds by clicking on goals from an IDE, being able to also click on errors there and get to the problematic code quickly, instead of fabricating a whole command line outside the IDE, manually selecting profiles etc. I left this hint behind for other developers who want to do the same in a convenient way, because while most people know `-DskipTests=true` and `-Dtest=MyTest`, many of them are not Maven Invoker gurus (myself included) and don't want to google the property name if once in 3 months an IT fails and they cannot remember. Please do not assume, that your level of deep Maven CLI knowledge is the same as everybody else's. It is just a comment and I left it there for the convenience of others as much as myself.

Reply:
by experience I know people will have such piece of code, forget to put it back within comments then commit/push.

Reply:
IT is CLI first. If the IDE cannot cope with it, I open a bug at jetbrains/eclipse.org etc.
If you run a maven goal from any IDE, you can add `-Dinvoker.test=testname` there, too.

Reply:
Fine, as you wish. I am going to remove it. *shrug*